compositor_render/src/wgpu/format/pixel_yuv_to_rgba.rs

```diff
@@ -0,0 +1,99 @@
+use crate::wgpu::{
+    common_pipeline::{Sampler, Vertex, PRIMITIVE_STATE},
+    texture::{PixelYuv422Texture, RGBATexture},
+};
+
+use super::WgpuCtx;
+
+#[derive(Debug)]
+pub struct PixelYuv422ToRgbaConverter {
+    pipeline: wgpu::RenderPipeline,
+    sampler: Sampler,
+}
+
+impl PixelYuv422ToRgbaConverter {
+    pub fn new(
+        device: &wgpu::Device,
+        yuv_textures_bind_group_layout: &wgpu::BindGroupLayout,
+    ) -> Self {
+        let shader_module =
+            device.create_shader_module(wgpu::include_wgsl!("pixel_yuv_to_rgba.wgsl"));
+        let sampler = Sampler::new(device);
+
+        let pipeline_layout = device.create_pipeline_layout(&wgpu::PipelineLayoutDescriptor {
+            label: Some("PixelYUV422 to RGBA color converter render pipeline layout"),
+            bind_group_layouts: &[yuv_textures_bind_group_layout, &sampler.bind_group_layout],
+            push_constant_ranges: &[],
+        });
+
+        let pipeline = device.create_render_pipeline(&wgpu::RenderPipelineDescriptor {
+            label: Some("Pixel YUV 4:2:2 to RGBA color converter render pipeline"),
```

Comment:
```suggestion
            label: Some("Interleaved YUV 4:2:2 to RGBA color converter render pipeline"),
```

Reply:
I would change pixel yuv to interleaved yuv everywhere

Reply:
I think I took that from here https://www.intel.com/content/www/us/en/docs/ipp/developer-guide-reference/2021-11/pixel-and-planar-image-formats.html

My main concern is that was potentially intended to maybe refer to all single plane format. Would it be correct to call RGBA or BGRA interleaved format?